practice/src/index.html

```diff
@@ -420,38 +420,36 @@ <h4 class="destination-title">Parau Mountain</h4>
     
     <!-- Discount section -->
     <section class="discount-section">
-      <div class="wrapper-discount">
-        <div class="container discount-main">
-          <div class="discount-body">
-            <hgroup class="text discount-heading">
-              <h2 class="main-title ">For beloved customers</h2>
-              <h3 class="secondary-title discount-title">20% discount if you have used our trip services!</h3>
-              <p class="discount-description description">Let's maximize your next holiday with us with the best experience.
-              </p>
-            </hgroup>
-            <div class="discount-btn">
-              <a href="javascrip:void(0)" class="btn btn-primary btn-claim">Claim Now</a>
-            </div>
+      <div class="container wrapper-discount">
+        <div class="discount-body">
+          <hgroup class="text discount-heading">
+            <h2 class="main-title">For beloved customers</h2>
+            <h3 class="secondary-title discount-title">20% discount if you have used our trip services!</h3>
+            <p class="discount-description description">Let's maximize your next holiday with us with the best experience.
+            </p>
+          </hgroup>
+          <div class="discount-btn">
```

Comment:
No need to wrap div here. You can use `<a href="javascrip:void(0)" class="btn btn-primary btn-claim">Claim Now</a>`. Because you have use display-inline for it